recipes/mdal/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "mdal" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lutraconsulting/MDAL/archive/0.7.2.tar.gz
+  sha256: 37c050ece0ad2cf0e04f8519f23a4125ab7f88f04f4989de8a2a5daaef846e02
+
+build:
+  number: 0
+  skip: false 
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+  host:
+    - libgdal
+  run:
+    - libgdal
```

Comment:
This [uses run_exports](https://github.com/conda-forge/gdal-feedstock/blob/b70b12acbfa93c4fafad7622ab3d2ca33de38737/recipe/meta.yaml#L65-L67) so no need to add it in `run`:

```suggestion
```